server.js

```diff
@@ -1,38 +1,70 @@
-import express from 'express'
-import bodyParser from 'body-parser'
-import cors from 'cors'
-import mongoose from 'mongoose'
-
-// If you're using one of our datasets, uncomment the appropriate import below
-// to get started!
-// 
-// import goldenGlobesData from './data/golden-globes.json'
-// import avocadoSalesData from './data/avocado-sales.json'
-// import booksData from './data/books.json'
-// import netflixData from './data/netflix-titles.json'
-// import topMusicData from './data/top-music.json'
-
-const mongoUrl = process.env.MONGO_URL || "mongodb://localhost/project-mongo"
-mongoose.connect(mongoUrl, { useNewUrlParser: true, useUnifiedTopology: true })
-mongoose.Promise = Promise
-
-// Defines the port the app will run on. Defaults to 8080, but can be 
-// overridden when starting the server. For example:
-//
-//   PORT=9000 npm start
-const port = process.env.PORT || 8080
-const app = express()
+import express from "express";
+import bodyParser from "body-parser";
+import cors from "cors";
+import mongoose from "mongoose";
+
+import booksData from "./data/books.json";
+
+//Author model and connection to database
+const mongoUrl = process.env.MONGO_URL || "mongodb://localhost/project-books";
+mongoose.connect(mongoUrl, { useNewUrlParser: true, useUnifiedTopology: true });
+mongoose.Promise = Promise;
+
+const port = process.env.PORT || 8080;
+const app = express();
+
+//A model
+const Author = mongoose.model("Author", {
+  title: String,
+  authors: String,
+  isbn: Number,
+});
 
 // Add middlewares to enable cors and json body parsing
-app.use(cors())
-app.use(bodyParser.json())
+app.use(cors());
+app.use(bodyParser.json());
+
+if (process.env.RESET_DATABASE) {
+  const seedDatabase = async () => {
+    await Author.deleteMany();
+
+    booksData.forEach((item) => {
+      const newAuthor = new Author(item);
+      newAuthor.save();
+    });
+  };
+  seedDatabase();
+}
+
+// Routes starts here
+app.get("/", (req, res) => {
+  res.send("Welcome, this is the database full of great authors and books");
+});
+
+//Get all authors
+app.get("/authors", async (req, res) => {
+  const allAuthors = await Author.find(req.query);
+  console.log(allAuthors);
```

Comment:
console.logs can be removed before submitting final project

Reply:
fixed!